lib/Controller/Stats.php

```diff
@@ -582,56 +583,70 @@ public function export()
         }
 
         // Get result set
-        $resultSet =  $this->timeSeriesStore->getStats([
-            'fromDt'=> $fromDt,
-            'toDt'=> $toDt,
+        $resultSet = $this->timeSeriesStore->getStats([
+            'fromDt' => $fromDt,
+            'toDt' => $toDt,
             'displayIds' => $displayIds,
         ]);
 
-        $out = fopen('php://output', 'w');
-        fputcsv($out, ['Stat Date', 'Type', 'FromDT', 'ToDT', 'Layout', 'Display', 'Media', 'Tag', 'Duration', 'Count', 'Engagements']);
+        // We only export if isExport is 1 else we return the count of stats
+        if (!empty($isExport)) {
```

Comment:
To get a count of stats we ought to have a separate endpoint to call - probably a web only route in routes-web

Having it triggered by a special parameter means that this method now has two different purposes, which is not RESTful.